package.json

```diff
@@ -98,7 +142,7 @@
     "inquirer": "8.2.5",
     "jest": "29.0.3",
     "jest-circus": "29.0.3",
-    "jest-cli": "29.5.0",
+    "jest-cli": "29.0.3",
```

Comment:
Not part of the dedupe per per-se... but this is creating a very fragile situation